nilearn/input_data/nifti_masker.py

```diff
@@ -31,8 +31,8 @@ def filter_and_mask(imgs, mask_img_, parameters,
                     memory_level=0, memory=Memory(cachedir=None),
                     verbose=0,
                     confounds=None,
-                    copy=True):
-
+                    copy=True,
+                    return_affine=True):
```

Comment:
Why this change? It add an argument that may trigger recomputation and doesn't add much to the function.